### ISSUE TITLE
Craft 4.0 upgade migration fails 

### ISSUE THREAD

**Issue by davola** [verbb/icon-picker#59]
### Describe the bug

When running the migrations after updating craft, the migration process fails when applying m221111_000000_modernize_model migration

### Steps to reproduce

1. update craft to v4
2. run `php craft migrate/all`
3. the migration `m221111_000000_modernize_model` will fail
4. no error loged neither shown on screen - just fails on the first UID found.

### Craft CMS version

4.0.3

### Plugin version

2.0.1, 2.0.2, 2.0.3

### Multi-site?

yes

### Additional context

This pnly fails when running on exisetnt DB backup.
It succeeds when running on empty database.

**Comment by engram-design** [comment]
What's the actual output of the error? There must be _some_ form of detail about this issue, whether on screen in the CLI or in the logs? If not, that seems pretty odd! What would be the `first UID found`?

**Comment by davola** [comment]
Hey @engram-design !
When craft v4 starts for the first time after updating from v3 and running `craft migrate/all` all the migrations starts, until it reaches the one from the icon-picker migration class `m221111_000000_modernize_model`

As you can see on the screenshot below when it tries to apply the migration on the first field ` matrixBlockType6ab89dc8-ace0-41e5-8225-090d40a814c9` (the suffix is an IconPicker field UID we have on the CMS) and it fails silently.
![image](https://user-images.githubusercontent.com/4193166/203873524-d7b3dc47-a664-46ba-9a7f-afca27613a6b.png)

By silentrly I mean there is no error on the console (as shown above), nor any logs with errors either. Just aborts the migration.

For your reference, the field `6ab89dc8-ace0-41e5-8225-090d40a814c9` configuration yml is as follows:
```
# config/project/matrixBlockTypes/icon--6ab89dc8-ace0-41e5-8225-090d40a814c9.yaml

field: d4171fe6-b9a7-4edb-b81d-df79414251b3 # Icon Or Image
fieldLayouts:
  463db78c-da95-49c2-a8a8-7e925cd04de4:
    tabs:
      -
        elements:
          -
            fieldUid: e3fb17ae-5624-483f-9a1b-9e09833c7de3 # Icon
            instructions: null
            label: null
            required: true
            tip: null
            type: craft\fieldlayoutelements\CustomField
            warning: null
            width: 50
          -
            fieldUid: 86cb3b47-dad6-422c-992b-37025511cfdf # Bubble Color
            instructions: null
            label: null
            required: true
            tip: null
            type: craft\fieldlayoutelements\CustomField
            warning: null
            width: 50
        name: Content
        sortOrder: 1
fields:
  86cb3b47-dad6-422c-992b-37025511cfdf: # Bubble Color
    columnSuffix: null
    contentColumnType: text
    fieldGroup: null
    handle: bubbleColor
    instructions: ''
    name: 'Bubble Color'
    searchable: false
    settings:
      default: null
      options: ''
      palette: Metronet
      useConfigFile: '1'
    translationKeyFormat: null
    translationMethod: none
    type: percipiolondon\colourswatches\fields\ColourSwatches
  e3fb17ae-5624-483f-9a1b-9e09833c7de3: # Icon
    columnSuffix: null
    contentColumnType: string
    fieldGroup: null
    handle: icon
    instructions: ''
    name: Icon
    searchable: false
    settings:
      columnType: text
      iconSets: '*'
      remoteSets: ''
      showLabels: '1'
    translationKeyFormat: null
    translationMethod: none
    type: verbb\iconpicker\fields\IconPickerField
handle: icon
name: 'Icon Bubble'
sortOrder: 1
```

**Comment by engram-design** [comment]
My goodness 🤦‍♂️ I left a debug statement in that migration which caused it to halt. Sorry about that.

Should be fixed in [2.0.5](https://github.com/verbb/icon-picker/releases/tag/2.0.5)

**Comment by davola** [comment]
lol yeah! Thanks a lot @engram-design for the quick response and solution!
